abs_x : 3
abs_y : 2
width : 4
height : 5
trans_vel : 2
rot_vel : 1
frequency : 100
